crates/bevy_reflect/src/serde/de/tuple_utils.rs

```diff
@@ -6,6 +6,7 @@ use crate::{
     DynamicTuple, TupleInfo, TupleStructInfo, TupleVariantInfo, TypeRegistration, TypeRegistry,
     UnnamedField,
 };
+use alloc::string::ToString;
```

Comment:
At the moment I've added this crate to our `no_std` compile-check CI task, which will catch everything _except_ macros which aren't explicitly used, only exported. I want to improve this story, but I think this will naturally resolve itself once other crates are `no_std` compatible (as they'll fail if the macros aren't working).